sync/src/types.rs

```diff
@@ -432,6 +437,33 @@ impl Peers {
     pub fn disconnected(&self, peer: PeerIndex) -> Option<PeerState> {
         self.state.write().remove(&peer)
     }
+
+    pub fn insert_unknown_list(&self, peer: PeerIndex, hash: Byte32) {
```

Comment:
The name is confusing, I suggest changing it to `insert_into_unknown_list` or `insert_unknown_block_hash`.